tools/ds-identify

```diff
@@ -869,7 +869,7 @@ dscheck_NoCloud() {
 
 is_ds_enabled() {
     local name="$1" pad=" ${DI_DSLIST} "
-    [ "${pad#* $name }" != "${pad}" ]
+    [ "${pad#* "$name" }" != "${pad}" ]
```

Comment:
@Again let's make variable expansion more conspicuous for readability, especially in these nested contexts

```suggestion
    [ "${pad#* "${name}" }" != "${pad}" ]
```